bat-utils/lib/runtime-kafka.js

```diff
@@ -57,25 +57,44 @@ class Kafka {
       // parallel processing on topic level
       const topicPromises = Object.keys(batchOfMessages).map(async (topic) => {
         // parallel processing on partition level
```

Comment:
This is no longer accurate, but we could switch back to processing the partitions in parallel - right? Since we're saving the offset on a (topic, partition) basis.

Reply:
that is probably correct